projects/azure/base-images/test-base/Dockerfile

```diff
@@ -0,0 +1,72 @@
+# This is development dockerfile
+# original copy is kept inside build-config
+FROM docker.io/centos:7
```

Comment:
Should this be rhel7?

Reply:
its our test image, it should but it will be hell of the problems when we try to install all our test tool into rhel :/